src/Relationships/HasOne.php

```diff
@@ -98,6 +112,7 @@ public function getRawValue()
 
 	public function setInjectedValue($value): bool
 	{
+		$this->isValueFromStorage = false;
```

Comment:
Is this correct? Alternatively should it be in set? (it is needed there because symmetric rel update)